src/models/test/user_test.ts

```diff
@@ -1,18 +1,14 @@
-import { expect } from "chai";
-import "mocha";
 import User from "./../user";
 
 /**
  * Unit test suite for the user model.
  */
-describe("User model", () => {
-  it("should be able to get its own ID", () => {
-    const user: User = new User(0, "username");
-    expect(user.id).to.equal(0);
-  });
+it("should be able to get its own ID", () => {

```

Comment:
Should this and the below unit test be combined to make "should be able to get its own ID and username"?

Reply:
Discussed offline